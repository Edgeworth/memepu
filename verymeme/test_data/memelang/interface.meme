// status: ok
// input: ''
// output: ''
// returns: 0
intf Meme<T> {
  fn testFunc() i8;
  fn testFunc2(a: *T) i16;
}
fn main() i8 { ret 0; }
