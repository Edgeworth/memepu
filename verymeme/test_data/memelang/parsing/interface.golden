File
 Intf
  FnSig
   Typename(testFunc)
   Type(i8)
  FnSig
   Typename(testFunc2)
   VarDecl
    VarRef(a)
    Type(T)
     Qualifier(const: 0 ptr: 1, array: 0)
   Type(i16)
